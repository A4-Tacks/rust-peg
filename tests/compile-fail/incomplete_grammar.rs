
peg::parser!();

peg::parser!(
    grammar parser() for str
);

fn main() {}