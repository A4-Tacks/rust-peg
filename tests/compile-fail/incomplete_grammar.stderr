error: expected one of "#", "grammar", "pub" at end of input
 --> tests/compile-fail/incomplete_grammar.rs:2:1
  |
2 | peg::parser!();
  | ^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `peg::parser` (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected one of "::", "<", "{" at end of input
 --> tests/compile-fail/incomplete_grammar.rs:4:1
  |
4 | / peg::parser!(
5 | |     grammar parser() for str
6 | | );
  | |_^
  |
  = note: this error originates in the macro `peg::parser` (in Nightly builds, run with -Z macro-backtrace for more info)
